src/main/java/org/opencmshispano/multimoduleimporter/CmsModuleMultiImportThread.java

```diff
@@ -197,15 +200,14 @@ public void run() {
 
     /**
      * 
-     * @param moduleName The name of the module to import
+     * @param moduleFilename The filename of the module to import
      * @param finished
      * @return true if module has been imported
      */
-    private boolean importModule(final String moduleName, final boolean finished) {
-        CmsModule m = moduleManager.getModule(moduleName);
+    private boolean importModule(final String moduleFilename, final boolean finished) {
+        CmsModule m = moduleManager.getModule(moduleFilename);
```

Comment:
I missed this test (`getModule(...) == null`). I'll see if I can retrofit this test in the next revision.